code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -226,10 +236,75 @@
 	else
 		return
 
+/mob/living/simple_animal/bot/medbot/proc/tip_over(mob/user)
+	playsound(src, 'sound/machines/warning-buzzer.ogg', 50)
+	user.visible_message("<span class='danger'>[user] tips over [src]!</span>", "<span class='danger'>You tip [src] over!</span>")
+	mode = BOT_TIPPED
+	var/matrix/mat = transform
+	transform = mat.Turn(180)
+	tipper_name = user.name

```

Comment:
I don't think this'll look good in a message if the person has a mask

Reply:
It gets whatever their visible name is at the time of tipping, I figured it'd be more immersion breaking if the medbot magically deduced your identity than it would be to call out that some unknown was bullying them. Plus this lets you disguise as someone else to tip the bot so that the other person gets reported for being a psycho

Reply:
I agree, I was more worried about them calling you out on a name like "john doe (disguised as bob stink)"

Reply:
If it's good enough for beepsky it's good enough for medbots, I'm pretty sure I do it the same way for the Orion Trail snitching too

Reply:
I guess so, lets hope it doesnt cause any weird behavior :)